src/config/test_config.c

```diff
@@ -96,8 +96,10 @@ static const WT_CONFIG_CHECK confchk_workload_generator_subconfigs[] = {
   {NULL, NULL, NULL, NULL, NULL, 0}};
 
 static const WT_CONFIG_CHECK confchk_workload_tracking_subconfigs[] = {
-  {"enabled", "boolean", NULL, NULL, NULL, 0}, {"op_rate", "string", NULL, NULL, NULL, 0},
-  {NULL, NULL, NULL, NULL, NULL, 0}};
+  {"enabled", "boolean", NULL, NULL, NULL, 0}, {"is_custom", "boolean", NULL, NULL, NULL, 0},
```

Comment:
I'm very much opposed to the use of "custom" to describe the validation components. The cppsuite in general is customizable. I feel like we are going in a weird direction with this.

We don't need to specify a boolean for any of the other methods we can override. Can we do something similar to how the `workload_generator` methods are defined? 

Also, we shouldn't start prefixing config keys with `is_`, none of the other keys use this style.

Reply:
We could go with `user_defined`?

Reply:
Ideally we could have a solution that does not involve setting a boolean at all. We don't do this when overriding the other database operations. We want this to be easy for the user.

Reply:
We can access `key_format` and `value_format` from our cursors. Perhaps we drop `is_custom`, and have a check inside the default validation that `op_track_cursor.key_format == "QSQ"` and `op_track_cursor.value_format == "iS"`. We have some similar checks already in different functions.

Reply:
A user could define those formats (and store other info) and still want a custom validation.

Reply:
Please ignore my previous comment, if we don't need to check whether the formats have been custom-defined then it won't be an issue.

Reply:
is_custom has been removed and replaced with the checks I described above in https://github.com/wiredtiger/wiredtiger/commit/325247ff2f0d2cdd6175e87c536d66b735e51806